resources/js/components/tiptap.js

```diff
@@ -91,7 +92,10 @@ export default function (content, searchModel = ['user', 'role']) {
                         },
                     },
                     onUpdate: ({ editor }) => {
-                        this.content = editor.getHTML();
+                        clearTimeout(this.timeout);
+                        this.timeout = setTimeout(() => {
+                            this.content = editor.getHTML();
+                        }, debounceDelay); // Delay Livewire updates by debounceDelay (300ms by default)
```

Comment:
comment says default is 300ms, but debounceDelay = 0 in given function. Am I missing something?